jsdocConfig.json

```diff
@@ -3,7 +3,7 @@
     "allowUnknownTags": true
   },
   "opts": {
-    "destination": "./dist/docs"
+    "destination": "./docs"
```

Comment:
Sorry, this was a work in progress. I was following Brian's setup instructions, so this was in flux for a bit.

I actually renamed this as "public" because it will have to contain both the jsDoc and examples published to GitHub pages. I chose "public" instead of "publish" because we have more than one publish script for pushing content to master and master-dist. It gets confusing using the term "publish" in multiple places.